### PR TITLE
XW-5119 | New Typography

### PR DESCRIPTION
**Pull request by bkoval**
@Wikia/x-wing

### CODE REVIEW

File: dist/scss/wds-typography/placeholders/_line-height-scale.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+%wds-line-height-4xl {
+	line-height: 52px;
+}
+
+%wds-line-height-paragraph {
+	line-height: 28px;
+}
+
+%wds-line-height-short-paragraph {
```
Comment (hakubo) [inline dist/scss/wds-typography/placeholders/_line-height-scale.scss]: it's the same as wds-line-height-paragraph
why do we need both?

File: tests/dummy/app/components/font-weight-line.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+	classNames: 'typescale-line'
```
Comment (hakubo) [inline tests/dummy/app/components/font-weight-line.js]: tagName: ''
and add the wrapping div to template
cleaner approach
one that will be soon default

File: dist/scss/wds-typography/placeholders/_text-style.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+%wds-text-transform-lowercase {
+	text-transform: lowercase;
+}
+
+%wds-text-transform-none {
+	text-transform: none;
+}
+
+%wds-midlight {
```
Comment (vforge) [inline dist/scss/wds-typography/placeholders/_text-style.scss]: It won't work as expected for multi-line, you need to add `box-decoration-break: clone;`
see https://github.com/Wikia/f2/search?q=box-decoration-break&unscoped_q=box-decoration-break
see https://developer.mozilla.org/en-US/docs/Web/CSS/box-decoration-break

Comment (rybmat) [reply dist/scss/wds-typography/placeholders/_text-style.scss]: @vforge we tested it and it works: 
screen shot 2018-07-25 at 15 47 23
the only case we are aware it does not work and `box-decoration-break: clone;` doesn't help either is when parent element uses `display: flex`

Comment (rybmat) [reply dist/scss/wds-typography/placeholders/_text-style.scss]: btw, we should use % in the gradient instead of px to support all font sizes

Comment (vforge) [reply dist/scss/wds-typography/placeholders/_text-style.scss]: @rybmat There's 3px padding on the left on the 1st line and not on the right side of the 1st line. There's no padding on the 2nd line and on the left side on the 3rd line. The right side on the 3rd line has a padding.

Comment (rybmat) [reply dist/scss/wds-typography/placeholders/_text-style.scss]: obviously I'm blind ;)

Comment (bkoval) [reply dist/scss/wds-typography/placeholders/_text-style.scss]: I added the rule, thanks @vforge

**Pull request merged**